Makefile

```diff
@@ -156,7 +161,7 @@ clean:
 	cargo clean
 
 package.zip: version-debug version-release
```

Comment:
note that the target name does no longer match the produced binary. Let's rename it to just `package` and add to phony target list

Reply:
Done